package.json

```diff
@@ -64,7 +64,8 @@
     "redux": "4.0.0",
     "redux-debounce": "1.0.1",
     "redux-thunk": "2.2.0",
-    "taskcluster-client-web": "7.0.0"
+    "taskcluster-client-web": "7.0.0",
+    "taskcluster-lib-scopes": "1.9.0"
```

Comment:
Could you add `taskcluster-lib-scopes` to the vendor package list in `neutrino-custom/base.js`, so it's moved to the vendor chunk? :-)

Currently it's being added to each entrypoint separately, and so with the rest of the PR ends up increasing the combined bundle size by 70KB compared to the [PR parent commit](https://travis-ci.org/mozilla/treeherder/jobs/373106507#L714) from `master`. (Our awful bundle sizes are something we're working on bringing down gradually)